mobify/progressive-web-scaffold
# Site Header

Style the Header, and add some other tweaks

 **JIRA**: [WEB-922](https://mobify.atlassian.net/browse/WEB-922)
## Changes
- Style the Site Header
- Add the correct SVG icons from the mockups
- Add a new build script for the SVG sprite (`npm run build-sprite`)
- Tweaks to various components: Button, HeaderBar, IconLabelButton, IconLabel, Icon, Image, NavItem, Sheet and SkipLinks
- Tweaks to various containers: App, Header and Navigation
## How to test-drive this PR
- `npm i`
- `npm run dev`
- Visit [here](https://preview.mobify.com/?url=http%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1) to preview Merlin's Potions
- Observe the site header and navigation
